DateTime.js

```diff
@@ -15,6 +15,12 @@ var viewModes = Object.freeze({
 	TIME: 'time',
 });
 
+var eventBoundaryEventTypes = Object.freeze([
```

Comment:
Maybe a future enhancement, but I could see people wanting to configure which of these this applies to if enabled?